tests/PactTestsSpec.hs

```diff
@@ -190,3 +193,24 @@ checkLegacyProdParser expectSuccess fp = describe fp $ do
       pc `shouldSatisfy` isLeft
  where
   parse = legacyParsePact <$> T.readFile fp
+
+-- Versioned natives tests
+
+versionedNativesTest :: ExecutionFlag -> [Text] -> SpecWith ()
+versionedNativesTest flag natives = do
+  let msg = "Successfully disables " <> show flag <> " natives"
+  it msg $ do
+    let rs = versionedNativesRefStore (mkExecutionConfig [flag])
+        rs' = versionedNativesRefStore (mkExecutionConfig [])
+        nativesDisabled = S.fromList natives
+    rs `shouldSatisfy` views rsNatives (S.disjoint nativesDisabled . S.fromList  . HM.keys)
+    rs' `shouldSatisfy` views rsNatives (S.isSubsetOf nativesDisabled . S.fromList  . HM.keys)
+
+versionedNativesTests :: SpecWith ()
+versionedNativesTests = describe "versionedNativesTests" $ do
+  versionedNativesTest FlagDisablePact40 ["enumerate" , "distinct" , "emit-event" , "concat" , "str-to-list"]
```

Comment:
Rather than repeating these lists of names, can we make the nameLists as `[String]` constant in Pact, and then use them in both locations?